packages/form-js-viewer/src/render/components/form-fields/FilePicker.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @returns {import("preact").JSX.Element}
+ */
+export function FilePicker() {
+  return null;
+}
+
+FilePicker.config = {
+  type: 'filepicker',
+  keyed: true,
+  label: 'File picker',
+  group: 'basic-input',
```

Comment:
Unsure about if this is basic, but for now doesn't matter much to me.

Reply:
On the mocks it's basic, in the end it will be just simple HTML file input but we can always change it easily